<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>160</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formL_1"/>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>160</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formL_5"/>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>160</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formL_2"/>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>160</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formL_3"/>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>160</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formL_4"/>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>160</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formL_6"/>
   </widget>
   <widget class="QPushButton" name="add_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
